From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
Date: Mon, 29 Aug 2022 08:43:24 +0200
Subject: [ARCHEOLOGY] Add patches needed for Pi-KVM on Rpi4 (#4127)

> X-Git-Archeology: > recovered message: > * Add patches needed for Pi-KVM on Rpi4
> X-Git-Archeology: > recovered message: > * Remove patch that does not belong here
> X-Git-Archeology: > recovered message: > * Update config
> X-Git-Archeology: > recovered message: > * Update edge config
> X-Git-Archeology: - Revision 2fb3aaaf86a7463ef1d26f4d623e1d3289ea4f25: https://github.com/armbian/build/commit/2fb3aaaf86a7463ef1d26f4d623e1d3289ea4f25
> X-Git-Archeology:   Date: Mon, 29 Aug 2022 08:43:24 +0200
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Add patches needed for Pi-KVM on Rpi4 (#4127)
> X-Git-Archeology:
> X-Git-Archeology: - Revision acf5810cbe38c7578907ecd06e61abb0a446df1e: https://github.com/armbian/build/commit/acf5810cbe38c7578907ecd06e61abb0a446df1e
> X-Git-Archeology:   Date: Sat, 11 Mar 2023 19:19:13 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igor@armbian.com>
> X-Git-Archeology:   Subject: Move Rpi kernels to 6.2, move current to legacy (#4898)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 5f2a1af2193187f0b994d6f163b94f370a01134f: https://github.com/armbian/build/commit/5f2a1af2193187f0b994d6f163b94f370a01134f
> X-Git-Archeology:   Date: Fri, 28 Apr 2023 21:32:52 +0200
> X-Git-Archeology:   From: Ricardo Pardini <ricardo@pardini.net>
> X-Git-Archeology:   Subject: `bcm2711`/`edge`: bump to `6.3` (`rpi4b`, RPi Foundation Kernel)
> X-Git-Archeology:
---
 drivers/usb/gadget/function/f_mass_storage.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/usb/gadget/function/f_mass_storage.c b/drivers/usb/gadget/function/f_mass_storage.c
index da07e45ae6df..723d70766a49 100644
--- a/drivers/usb/gadget/function/f_mass_storage.c
+++ b/drivers/usb/gadget/function/f_mass_storage.c
@@ -2967,11 +2967,11 @@ void fsg_common_set_inquiry_string(struct fsg_common *common, const char *vn,
 	/* Prepare inquiryString */
 	i = get_default_bcdDevice();
 	snprintf(common->inquiry_string, sizeof(common->inquiry_string),
-		 "%-8s%-16s%04x", vn ?: "Linux",
+		 "%-8s%-16s%04x", vn ?: "PiKVM",
 		 /* Assume product name dependent on the first LUN */
 		 pn ?: ((*common->luns)->cdrom
-		     ? "File-CD Gadget"
-		     : "File-Stor Gadget"),
+		     ? "CD-ROM Drive"
+		     : "Flash Drive"),
 		 i);
 }
 EXPORT_SYMBOL_GPL(fsg_common_set_inquiry_string);
-- 
Armbian

